tests/utils/test_xml_helpers.py

```
@@ -0,0 +1,65 @@
+import unittest
+import xml.etree.ElementTree as ET
```

Ruff (0.8.0)

`xml.etree.ElementTree` imported but unused

Remove unused import: `xml.etree.ElementTree`

(F401)